ibis/pandas/tests/test_udf.py

```diff
@@ -35,6 +39,10 @@ def add_one(x):
     def times_two(x, scope=None):
         return x * 2.0
 
+    @udaf(input_type=[dt.double], output_type=dt.double, analytic=True)
```

Comment:
@kszucs What do you think about the name of this flag, `analytic`? I'm not sold on it, but I'm struggling to come up with a good name for it.

Reply:
I guess the issue is You want to use UDF naming convention for "User defined operations":
- `udf` for ValueOp operations
- `udaf` for Reduction and AnalyticalOp

```python
import ibis.expr.udf as udf

@udf.value
@udaf.reduction
@udaf.analytical
```

Eehhh, rather create an issue to keep struggling :)

Reply:
how about:

```python
@udf.elementwise
@udf.reduction
@udf.analytic
```

Reply:
Yes, that looks pretty good.